app/konnect/updates.md

```diff
@@ -9,6 +9,15 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
+## November 2022
+
+### 2022.11.xx
+
+**New Analytics predefined roles**
```

Comment:
Very nuanced but looking at a previous release note for Org Admin (read-only) we used `teams` instead of `roles`. Should we stay consistent with that?